install_config/install/advanced_install.adoc

```diff
@@ -95,6 +114,19 @@ this when using an interface that is not configured with the default route.
 |This variable overrides the system's public IP address. Use this for cloud
 installations, or for hosts on networks using a network address translation
 (NAT).
+
+|`*containerized*`
+|If set to *true*, containerized OpenShift services are run on the target host
+instead of installed using RPM packages. If set to *false* or unset, the default
+RPM method is used. RHEL Atomic Host requires the containerized method, and is
+automatically selected for you based on the detection of the
+*_/run/ostree-booted_* file. See
+link:../../install_config/install/rpm_vs_containerized.html[RPM vs
+Containerized] for more details.
+ifdef::openshift-enterprise[]
+Containerized installations are supported starting in OSE 3.1.1, and are
```

Comment:
I'd check with Mike, but I'd say we don't mention tech preview in 3.1.0 just say that it's supported starting with 3.1.1.